src/suites/cts/validation/createTexture.spec.ts

```diff
@@ -0,0 +1,155 @@
+export const description = `
+createTexture validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+class F extends ValidationTest {
+  getDescriptor(
+    options: {
+      width?: number;
+      height?: number;
+      arrayLayerCount?: number;
+      mipLevelCount?: number;
+      sampleCount?: number;
+      format?: GPUTextureFormat;
+    } = {}
+  ): GPUTextureDescriptor {
+    const {
+      width = 32,
+      height = 32,
+      arrayLayerCount = 1,
+      mipLevelCount = 1,
+      sampleCount = 1,
+      format = 'rgba8unorm',
+    } = options;
+    return {
+      size: { width, height, depth: 1 },
+      arrayLayerCount,
+      mipLevelCount,
+      sampleCount,
+      dimension: '2d',
+      format,
+      usage: GPUTextureUsage.OUTPUT_ATTACHMENT | GPUTextureUsage.SAMPLED,
+    };
+  }
+}
+
+export const g = new TestGroup(F);
+
+g.test('validation of sampleCount', async t => {
+  const { sampleCount, mipLevelCount, arrayLayerCount, success } = t.params;
+
+  const descriptor = t.getDescriptor({ sampleCount, mipLevelCount, arrayLayerCount });
+
+  await t.expectValidationError(() => {
+    t.device.createTexture(descriptor);
+  }, !success);
+}).params([
+  { sampleCount: 1, success: true }, // sampleCount of 1 is allowed
+  { sampleCount: 4, success: true }, // sampleCount of 4 is allowed
+  { sampleCount: 3, success: false }, // sampleCount of 3 is not allowed
+  { sampleCount: 4, mipLevelCount: 2, success: false }, // it is an error to create a multisampled texture with mipLevelCount > 1.
+  { sampleCount: 4, arrayLayerCount: 2, success: false }, // TODO: Remove when Chrome supports multisampled 2D array textures.
```

Comment:
Instead we should say success: true, and allow Chrome to be failing the test.

Reply:
Done